app/views/ad_templates/sponsored-text/show.html.erb

```diff
@@ -2,7 +2,7 @@
   <ins id="cf">
     <span class="cf-wrapper">
       Sponsored:&nbsp;
-      <%= image_tag @creative.icon_image.cloudfront_url, class: "cf-img", style: "width: 20px; height: 20px;", alt: @creative.name rescue nil %>
+      <%= image_tag @creative.icon_image.cloudfront_url, height: 20, width: 20, class: "cf-img", style: "width: 20px; height: 20px;", alt: @creative.name rescue nil %>
```

Comment:
When you pass the width and height attributes, you should remove the properties from `style`. IMO all images should have width and height attributes so that browsers can calculate their dimensions and avoid reflows